services/graphql-server/src/graphql/resolvers/platform/asset.js

```diff
@@ -14,4 +18,32 @@ module.exports = {
     alt: image => createAltFor(image),
     caption: image => createCaptionFor(image.caption),
   },
+
+  /**
+   *
+   */
+  Mutation: {
+    createAssetImageFromUrl: async (_, { input }, { base4rest, basedb }, info) => {
+      validateRest(base4rest);
```

Comment:
Should require auth

Reply:
`@requiresAuth` is present on the [definition](https://github.com/base-cms/base-cms/pull/276/files#diff-947cb5f9bc8aecc48b77fd9fbf3ce289R10)